### DISCUSSION TITLE
Bulk-add already existing users to some team via the weblate shell ?

### DISCUSSION THREAD

**Discussion by arbaes** [WeblateOrg/weblate#11963] [Q&A] [answered]
I wanted to restrict the default user group to specific languages, but it turns out we can't do that, so I was searching for a way to move those users in a specific groups with the right privileges.

While reading the official documentation here: https://docs.weblate.org/en/latest/admin/access.html#automatic-team-assignments
I found this note:

> As for now, there is no way to bulk-add already existing users to some team via the user interface. For that, you may resort to using the [REST API](https://docs.weblate.org/en/latest/api.html#api).

And I was wondering if I could use the weblate shell instead and do something like:

```
docker-compose exec --user weblate weblate weblate shell -c shell -c 'from weblate.trans.models import Users; Users.objects.set(group="NewGroup")```
```

Any help/advise would be greatly appreciated

**Comment by nijel** [comment, marked as answer]
Yes, shell should work as well, something like this should work:

```
docker-compose exec --user weblate weblate weblate shell -c shell -c 'from weblate.trans.models import User, Group; users = User.objects.filter(groups__name="OldGroup"); new_group = Group.objects.get(name="NewGroup"); for user in users: user.groups.set([new_group])```
```

**Reply by arbaes** [reply]
Thank you very much !
